url/resources/setters_tests.json

```diff
@@ -269,6 +269,16 @@
                 "protocol": "https:",
                 "port": ""
             }
+        },
+        {
+            "comment": "Tab and newline are stripped",
+            "href": "http://test/",
+            "new_value": "h\u000A\u000Att\u0009ps",
+            "expected": {
+              "href": "https://test/",
+              "protocol": "https:",
+              "port": ""
+            }
```

Comment:
It might be good to add one test for each of the other ASCII whitespace requiring that they cause failures. Both here and in location-protocol-setter.html.